### PR TITLE
Windows 1809 Support Alpha

### PR DESCRIPTION
**Pull request by thxCode**
- Adjust tabs to spaces in Dockerfiles
- Support Windows 1809
- Upgrade Flannel & CNI
- Adjust scripts
- Remove Nginx proxy

**Issues**:
https://github.com/rancher/rancher/issues/19173

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -10,9 +10,6 @@
         "v1.9.7"
     ],
     "flannel-cni": [
-        "v0.0.1"
-    ],
-    "nginx-proxy": [
-        "v0.0.1"
+        "v0.0.2"
     ]
```
Comment (loganhz) [inline VERSION:10]: Do we still need to remove this if we don't remove nginx proxy in old version?

Comment (thxCode) [reply VERSION:10]: I am going to use host running nginx instead of container running to avoid https://github.com/rancher/rancher/issues/16074

**Comment by loganhz** [comment]
LGTM

**Comment by loganhz** [comment]
As this is a breaking change, we need to checkout current master into 2.2 branch first

**Comment by thxCode** [comment]
@loganhz , this will not block the releasing for `1803`. After merging this, we can setup an `1809` CI environment with Hyper-V, and then execute `hooks/build.ps1` with `-PowerShellVersion nanoserver-1803`, for example:
```powershell
cd kubernetes-windows-dockerfiles
./hooks/build.ps1 -PowerShellVersion nanoserver-1803
```

The above script can package `1803` with Hyper-V container. The new built packages will have:
```
hyperkube:vx.y.z-nanoserver-1803
flannel-cni:v0.2.0-nanoserver-1803
```

File: scripts/image-clean.ps1 [hunk 2]
```diff
@@ -1,5 +1,8 @@
 Write-Host ('Remove all Docker containers ...')
 docker rm -f $(docker ps -aq);
 
+Write-Host ('Prune all container data ...')
```
Comment (orangedeng) [inline scripts/image-clean.ps1:4]: When do we use this script. It seems very dangerous.

**Comment by orangedeng** [comment]
LGTM

File: scripts/image-clean.ps1 (hunk 2 above)
Comment (thxCode) [reply scripts/image-clean.ps1:4]: There are some tools in `scripts` dir. They are only used for testing.

**Comment by thxCode** [comment]
Fixing is including in https://github.com/rancher/kubernetes-windows-dockerfiles/pull/11, close it.

**Pull request closed without merging**